bootstrap.py

```diff
@@ -765,6 +772,8 @@ class MEOptions:
         options.skip.append('puppet')
     if not options.removepkgs:
         options.skip.append('remove-obsolete-packages')
+    if options.new_capsule:
```

Comment:
This should be done after `get_bootstrap_rpm()` via `API_PORT = get_api_port()`. Also this makes moving *to* the internal capsules impossible.